creature/teqqqie; Weird Flavor Creatures.json

```diff
@@ -0,0 +1,5636 @@
+					],
+					"type": "variant"
+				}
+			],
+			"environment": [
+				"feywild",
+				"forest"
+			],
+			"fluff": {
+				"entries": [
+					"Deepstars are generally good creatures. They are gentle natured and friendly. They are dangerous only when they swarm, and they only swarm in response to fire. This is both a survival mechanism to compensate for their natural flammability, and a result of their desire to protect nature. During natural forest fires, deepstar swarms help guide and corral frightened creatures to safe caves and burrows, though the swarm will still panic if it catches fire itself. Generally, sighting a deepstar is considered a sign of good luck.",
+					"However, in areas ravaged by certain natural disasters (such as extreme fires or volcanic eruptions), or more commonly, industrial progress, a much more dangerous and malicious form of the deepstar may be found. In these areas where they are left with no more nature to retreat to or defend, deepstars go mad. Their innate drive to protect nature is twisted into a manic aggression toward any creature potentially responsible for the destruction of their home.",
+					"Corrupted deepstars, as they are known, are never encountered alone. They roam the desolate remains of their sanctuary in enraged swarms, attacking humanoids and nonnative creatures on sight. In addition to the damage to their psyches, corrupted deepstars suffer physical changes when they are deprived of nature. They lose the ability to produce poison, as well as their characteristic susceptibility to flames. While they are still flammable, it is ill advised to attempt to use fire against a corrupted deepstar swarm, since they will burst into flames and become even more dangerous.",
+					"Some druids, wizards, and other scholars have captured individuals from such swarms in an attempt to find a way to reverse their corruption, but none have yet succeeded."
+				],
+				"images": [
+					{
+						"type": "image",
+						"href": {
+							"type": "external",
+							"url": "https://i.imgur.com/xTsLcwY.png?1"
+						}
+					},
+					{
+						"type": "image",
+						"href": {
+							"type": "external",
+							"url": "https://i.imgur.com/ouLCdl9.png"
+						}
+					}
+				]
+			},
+			"tokenUrl": "https://i.imgur.com/YddHg99.png",
+			"senseTags": [
+				"B"
+			],
+			"actionTags": [
+				"Multiattack"
+			],
+			"languageTags": [
+				"CS",
+				"S"
+			],
+			"damageTags": [
+				"C",
+				"F",
+				"L",
+				"N",
+				"P"
+			],
+			"miscTags": [
+				"AOE",
+				"MW",
+				"RW"
+			]
+		},
```

Comment:
```suggestion
```

Reply:
Sorry, I'm new to github, what is changing here?

Reply:
if you check the `Checks` tab at the top of the PR, you can see [what it's failing on](https://github.com/TheGiddyLimit/homebrew/pull/1262/checks)

which is, you've got two copies of the exact same monster; this change is to delete one of those copies